apps/netutils/libcoap/LICENSE.GPL

```diff
@@ -0,0 +1,280 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Due to the license problem, we need to get comments from our opensource group. 
If libcoap is only available for GPL license, we cannot add it.